RfidShelf/RfidShelf.ino

```diff
@@ -499,6 +500,20 @@ void playFile() {
   }
 }
 
+void playFilename(String filepath) {
+  // IO takes time, reset watchdog timer so it does not kill us
+  ESP.wdtFeed();
+  Serial.print(F("Playing "));
+  Serial.println(filepath);
+  playing = PLAYING_FILE;
+  currentFile = filepath;
```

Comment:
This doesn't work, because currentFile contains the path now, not only the filename